src/tasks/AndroidAppBuilder/Templates/monodroid-librarymode.c

```diff
@@ -0,0 +1,63 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include <stdlib.h>
+#include <stdbool.h>
+#include <errno.h>
+#include <string.h>
+#include <jni.h>
+#include <android/log.h>
+#include <assert.h>
+#include <unistd.h>
+
+#define LOG_INFO(fmt, ...) __android_log_print(ANDROID_LOG_DEBUG, "DOTNET", fmt, ##__VA_ARGS__)
+#define LOG_ERROR(fmt, ...) __android_log_print(ANDROID_LOG_ERROR, "DOTNET", fmt, ##__VA_ARGS__)
```

Comment:
We dont need this here do we? nor do we need the `android/log.h` include

Reply:
No, removed